File: Backend/src/main/i5/las2peer/services/immersiveProjectManagementService/i5/las2peer/services/immersiveProjectManagementService/dataModel/gitHub/GitHubLabel.java
package i5.las2peer.services.immersiveProjectManagementService.i5.las2peer.services.immersiveProjectManagementService.dataModel.gitHub;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

/**
 * Created by bened on 18.07.2019.
 */
@JsonIgnoreProperties(ignoreUnknown = true)
public class GitHubLabel {
    private int id;
    private String nodeId;
    private String name;
    private String description;
    private String color;

    public int getId() {
        return id;
    }

    public String getNodeId() {
        return nodeId;
    }

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }

    public String getColor() {
        return color;
    }
}

File: Frontend/VIAProMa/Assets/Scripts/DataModel/GitHub/GitHubLabel.cs
using i5.VIAProMa.UI.ListView.Core;

namespace i5.VIAProMa.DataModel.GitHub
{
    /// <summary>
    /// Represents a requirement in the Requirements Bazaar
    /// </summary>
    public class GitHubLabel : IListViewItemData
    {
        private long id;
        private string nodeId;
        private string name;
        private string description;
        private string color;

        public long getId()
        {
            return id;
        }

        public string getNodeId()
        {
            return nodeId;
        }

        public string getName()
        {
            return name;
        }

        public string getDescription()
        {
            return description;
        }

        public string getColor()
        {
            return color;
        }
    }
}